src/app/technical-record/notes/notes.component.html

```diff
@@ -16,7 +13,10 @@
     </dd>
   </div>
 
-  <div class="govuk-summary-list__row" *ngIf="activeRecord.vehicleType === 'psv'">
+  <div
+    class="govuk-summary-list__row"
```

Comment:
unit tests same as above. one test to check for hgv/trl and one for psv should do the job.